src/pipelines/translation/translation_pipeline.rs

```diff
@@ -557,6 +1118,33 @@ impl TranslationOption {
             ModelType::M2M100 => Ok(TranslationOption::M2M100(M2M100Generator::new(
                 config.into(),
             )?)),
+            ModelType::NLLB => {
+                let config: GenerateConfig = config.into();
+                let tokenizer = TokenizerOption::from_file(
+                    ModelType::NLLB,
+                    config.vocab_resource.get_local_path()?.to_str().unwrap(),
+                    Some(
+                        &config
+                            .merges_resource
+                            .as_ref()
+                            .ok_or_else(|| {
+                                RustBertError::InvalidConfigurationError(
+                                    "M2M100 expects a merges resources to be provided".to_string(),
+                                )
+                            })?
+                            .get_local_path()?
+                            .display()
```

Comment:
could `to_str().unwrap()` be used for consistency with the path above?

Reply:
Done.